### PR TITLE
Fix copyright headers, move copyright owner from Resources.java to NOTICE

### PR DESCRIPTION
**Pull request by vlsi**
Copyright owner should be moved from header to NOTICE as per https://www.apache.org/legal/src-headers.html#headers

### CODE REVIEW

File: core/src/main/java/org/apache/calcite/runtime/Resources.java [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /*
- * Licensed to Julian Hyde under one or more contributor license
```
Comment (vlsi) [inline core/src/main/java/org/apache/calcite/runtime/Resources.java]: @julianhyde , AFAIK, we must not retain the original copyright owners in the source file, rather we move them to the NOTICE file (see https://www.apache.org/legal/src-headers.html#headers )

**Comment by julianhyde** [comment]
The link you cite is "SOURCE FILE HEADERS FOR CODE DEVELOPED AT THE ASF", does not apply here.

File: core/src/main/java/org/apache/calcite/runtime/Resources.java (hunk 1 above)
Comment (julianhyde) [reply core/src/main/java/org/apache/calcite/runtime/Resources.java]: A few lines down it says "Do not modify or remove any copyright notices or licenses within third-party works.".

Comment (vlsi) [reply core/src/main/java/org/apache/calcite/runtime/Resources.java]: Thanks for the pointer, and I kept the headers. I do not like how "source-included" file complicates build/edit operations as different files might happen to have different code styles, headers, etc, etc.

**Pull request merged**